Assets/Scripts/DTerrain/Model/Range.cs

```diff
@@ -52,6 +53,11 @@ public override bool Equals(object obj)
             }
         }
 
+        public override int GetHashCode()
```

Comment:
Why did we need this?

Reply:
I was getting warnings in Unity, apparently if you override the equals operator you also have to do this. And then figured min+max is a good enough key